packages/plugin-rsc/src/plugin.ts

```diff
@@ -1621,7 +1621,7 @@ export function vitePluginRscCss(
       }
       for (const next of mod?.importedModules ?? []) {
         if (next.id) {
-          if (isCSSRequest(next.id)) {
+          if (next.type === 'css') {
```

Comment:
It looks like css is actually `type: 'js'` :thinking: 

```
EnvironmentModuleNode {
  environment: 'rsc',
  url: '/src/index.css',
  id: '/home/hiroshi/code/others/vite-plugin-react/packages/plugin-rsc/examples/starter/src/index.css',
  file: '/home/hiroshi/code/others/vite-plugin-react/packages/plugin-rsc/examples/starter/src/index.css',
  type: 'js',
  info: undefined,
  meta: undefined,
  importers: Set(1) {
    EnvironmentModuleNode {
      environment: 'rsc',
      url: '/src/root.tsx',
      id: '/home/hiroshi/code/others/vite-plugin-react/packages/plugin-rsc/examples/starter/src/root.tsx',
      file: '/home/hiroshi/code/others/vite-plugin-react/packages/plugin-rsc/examples/starter/src/root.tsx',
      type: 'js',
```

Reply:
This is because `type: 'css'` is only for `.css?direct` request https://github.com/vitejs/vite/blob/95927d9c0ba1cb0b3bd8c900f039c099f8e29f90/packages/vite/src/node/server/moduleGraph.ts#L77